src/influencetx/schema.py

```diff
@@ -172,158 +256,214 @@ class Meta:
         connection_class = ExtendedConnection
 
     pk = graphene.Int()
+
     def resolve_pk(self, info, **kwargs):
         return self.id
-    
+
+    financial_disclosures = graphene.List(FinancialDisclosureType)
+
+    def resolve_financial_disclosures(self, info, **kwargs):
+        return FinancialDisclosure.objects.prefetch_related('jobs').prefetch_related('gifts').prefetch_related('stocks').prefetch_related('boards').prefetch_related('jobs').filter(legislator=self.pk)
```

Comment:
this is the only other interesting line, fixing the n+1 queries that Graphene defaults get you with. If you want to check the number of queries made, you can see in the network tab, in the response under _debug
<img width="1490" alt="Screen Shot 2020-08-16 at 6 34 02 PM" src="https://user-images.githubusercontent.com/2788860/90346512-1a524b00-dfef-11ea-977c-b1eeb14efe34.png">
As you can see this page now makes at most 9 SQL queries, which isn't perfect, but not bad either.